README.md

```diff
@@ -4,11 +4,53 @@
 
 NGINX Kubernetes Gateway is an open-source project that provides an implementation of the [Gateway API](https://gateway-api.sigs.k8s.io/) using [NGINX](https://nginx.org/) as the data plane. The goal of this project is to implement the core Gateway APIs -- `Gateway`, `GatewayClass`, `HTTPRoute`, `TCPRoute`, `TLSRoute`, and `UDPRoute` -- to configure an HTTP or TCP/UDP load balancer, reverse-proxy, or API gateway for applications running on Kubernetes. NGINX Kubernetes Gateway is currently under development and supports a subset of the Gateway API.
 
-> Warning: This project is actively in development (pre-alpha feature state) and should not be deployed in a production environment.
+> Warning: This project is actively in development (beta feature state) and should not be deployed in a production environment.
 > All APIs, SDKs, designs, and packages are subject to change.
 
-## Run NGINX Kubernetes Gateway
+## Getting Started
 
-1. [Build](docs/building-the-image.md) the NGINX Kubernetes Gateway container image.
-2. [Install](docs/installation.md) NGINX Kubernetes Gateway.
-3. Deploy various [examples](examples). 
+1. [Quick Start on a kind cluster](docs/running-on-kind.md).
+2. [Build](docs/building-the-image.md) the NGINX Kubernetes Gateway container image.
```

Comment:
sorry just noticed this, should we add a note that the build step can be skipped if they use the docker image from ghcr.io?

Reply:
Good callout. 

I changed up the link order and the build description. Let me know what you think